Extremem/src/main/java/com/amazon/corretto/benchmark/extremem/Report.java

```diff
@@ -120,6 +120,20 @@ static void output(String s1, String s2, String s3, String s4, String s5,
     out.println(s9);
   }
 
+  static void output(String s1, String s2, String s3, String s4, String s5,
```

Comment:
Why not use `varargs` for this `output` API?

Reply:
In original design (implementation), I was trying to avoid implicit object allocations, so that we could track exactly what memory is allocated by each thread.  I never finished that very detailed and laborious process of tracking allocations per thread.  Probably never will.  But this is why some of the peculiar APIs.

Reply:
Could we use JMX's [getThreadAllocatedBytes](https://docs.oracle.com/en/java/javase/11/docs/api/jdk.management/com/sun/management/ThreadMXBean.html#getThreadAllocatedBytes(long))?